File: challenges/4-fizz-buzz/python/virusxdrj/fizz_buzz.py
from sys import argv


def fizz_buzz(r: int) -> None:
    for i in range(1, r + 1):
        match i:
            case _ if (i % 5) == 0 and (i % 3) == 0:
                print("FizzBuzz")
                continue
            case _ if (i % 5) == 0:
                print("Buzz")
                continue
            case _ if (i % 3) == 0:
                print("Fizz")
                continue
        print(i)


if __name__ == "__main__":
    fizz_buzz(int(argv[1]))

File: challenges/4-fizz-buzz/typescript/clerijr/fizzBuzz.ts
let arg_x: number = Number(process.argv.slice(2, 3))

for(let i: number = 1; i <= arg_x; i++) {
    let message: string = i.toString()
    if(i%3 == 0 && i%5 != 0) {message = 'Fizz'}
    if(i%3 != 0 && i%5 == 0) {message = 'Buzz'}
    if(i%3 == 0 && i%5 == 0) {message = 'FizzBuzz'}
    console.log(message)
}